### PR TITLE
Add dashboards and auto gen in CI

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

fix: https://github.com/grafana/jsonnet-libs/issues/229

### CODE REVIEW

**Comment by yeya24** [comment]
Oh, CI failed... Will take a look soon...

**Comment by tomwilkie** [comment]
@jdbaldry can you pick this up please?  I really like the idea, we should make this happen (similar to https://github.com/grafana/cortex-jsonnet/blob/master/Makefile#L23)

**Comment by jdbaldry** [comment]
Sure thing, I've assigned myself and will take a look this afternoon or if not, early tomorrow :)

**Comment by yeya24** [comment]
Thanks for the review @jdbaldry, sorry for the delay of this pr. I will work on it this weekend.

**Comment by jdbaldry** [comment]
Hey @yeya24, I didn't realize that the CircleCI artifacts are not actually available to anybody but the maintainers. With that in mind, it seems like committing the generated JSON is probably the way we need to go. Sorry to flip-flop on you!

It would still be nice to buid that CI around the Makefile if that's cool. That should resolve the CI failure too as that relates to the fact the pipeline shell is `sh` and I'm guessing you are probably using `bash` or something that supports arrays.

File: consul-mixin/consul.json [hunk 1]
```diff
@@ -0,0 +1,710 @@
+{
```
[the remark this answers is not in the document]
Comment (jdbaldry) [reply consul-mixin/consul.json:1]: Scratch that, CircleCI artificacts are not publically available, so version controlling the JSON will be fine.

**Comment by jdbaldry** [comment]
Hey @yeya24. Long time no speak :D

Would you be amenable to making the suggested changes and I can look to get this merged and closed?

File: scripts/dashboard-gen.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
```
Comment (jdbaldry) [inline scripts/dashboard-gen.sh:1]: Although the shebang says `sh` I believe that this is probably using bash or another shell on your system which supports arrays. The POSIX shell doesn't have this support and is the reason this script is throwing an error in CI. Changing the shell to `bash` is less portable but should fix the error.
```suggestion
#!/usr/bin/env bash
```

**Comment by yeya24** [comment]
I am sorry I cannot continue this. Help wanted!

**Pull request closed without merging**